src/scope/repositories/sources.ts

```diff
@@ -182,8 +182,7 @@ to quickly fix the issue, please delete the object at "${this.objects().objectPa
     flattenedTesterDependencies,
     specsResults
   }: {
-    // @ts-ignore AUTO-ADDED-AFTER-MIGRATION-PLEASE-FIX!
-    consumerComponent: $ReadOnly<ConsumerComponent>;
+    consumerComponent: ConsumerComponent;
```

Comment:
@qballer , why did you remove the ReadOnly? it's deliberately there.

Reply:
It isn't respected in the function, and ReadOnly breaks. $ReadOnly is no good here.

Reply:
@qballer , The idea behind it was to not change anything in the parameter instance. All changes must be done on a clone of the parameter.
So, the syntax you see here is from Flow. The Typescript syntax is different, but it's still possible.
You can change the entire line to:
`readonly consumerComponent: ConsumerComponent;`
In general, you can see this great reference of the flow vs typescript in terms of syntax changes: https://github.com/niieani/typescript-vs-flowtype

Reply:
I swear something violated it. made it readonly.